Tests/DatadogTests/Datadog/RUM/RUMMonitor/Scopes/RUMViewScopeTests.swift

```diff
@@ -1059,7 +1059,7 @@ class RUMViewScopeTests: XCTestCase {
         XCTAssertEqual(error.error.source, .source)
         XCTAssertEqual(error.error.sourceType, .ios)
         XCTAssertNil(error.error.stack)
-        XCTAssertNil(error.error.isCrash)
+        XCTAssertTrue(error.error.isCrash == false)
```

Comment:
/nit
```suggestion
        XCTAssertFalse(error.error.isCrash)
```

Reply:
Of course! I think I was frantically fighting with linter :D